discord_thread_summarizer_using_openai/main.bal

```diff
@@ -0,0 +1,79 @@
+import ballerina/io;
+import ballerina/time;
+
+const string DISCORD_API_URL = "https://discord.com/api/v9";
+const TYPE_FORUM_CHANEL = 15;
+
+configurable string botToken = ?;
+configurable int forumChannelId = ?;
```

Comment:
Instead of spreading const values and configurable variables into multiple files, I suggest we should group them together into single places for easy maintenance and readability.  
Another pattern is to define them where it is used if only used within a single file. But don't mix both patterns. Look at the usage of the `botToken` configurable.

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/69/commits/1dbf76e067fd08cc0430182aa02b7c98d154f3ac